Uml/Uml.csproj

```diff
@@ -5,13 +5,19 @@
     <Company>RHEA System S.A.</Company>
     <Title>UML for dot net</Title>
     <Version>0.1.0</Version>
+    <AssemblyVersion>0.1.0</AssemblyVersion>

```

Comment:
i propose we remove AssemblyVersion and FileVersion, see https://andrewlock.net/version-vs-versionsuffix-vs-packageversion-what-do-they-all-mean/ for a decent explanation